app/components/views/Buy/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { providers } from "ethers";
+import { ImageCard } from "../../ImageCard";
+import * as styles from "./styles";
+import { ButtonPrimary, Text } from "@klimadao/lib/components";
+import { Trans } from "@lingui/macro";
+import Payment from "@mui/icons-material/Payment";
+
+interface Props {
+  provider: providers.JsonRpcProvider;
+  address?: string;
+  isConnected: boolean;
+  loadWeb3Modal: () => void;
+}
+
+export const Buy = (props: Props) => {
+  return (
+    <>
+      <div className={styles.buyCard}>
+        <div className={styles.buyCard_header}>
+          <Text t="h4" className={styles.buyCard_header_title}>
+            <Payment />
+            <Trans id="buy.buy_klima">Buy KLIMA</Trans>
+          </Text>
+          <Text t="caption" color="lightest">
+            <Trans id="buy.how_to_buy" comment="Long sentence">
+              We've partnered with Mobilum so you can purchase Klima with a
+              credit card in just a few clicks. After your purchase is complete,
+              be sure to stake your KLIMA to start earning rewards. You may need
+              to refresh the page to see your new KLIMA balance.
+            </Trans>
+          </Text>
+        </div>
+        {props.isConnected && (
```

Comment:
It was quite slow to load after I clicked connect wallet, even on my top-tier internet connection. So there were a few confusing seconds of blank screen.

Can we mitigate this? A few ideas:
- render and load the iframe but hide it with css until a wallet is connected?
- put a loading spinner underneath that gets covered up when the iframe is done loading?
Or are there some sort of native iframe loading hooks we can use to show a spinner/placeholder?